script/tool/lib/src/common/plugin_command.dart

```diff
@@ -14,15 +14,16 @@ import 'package:yaml/yaml.dart';
 import 'core.dart';
 import 'git_version_finder.dart';
 import 'process_runner.dart';
+import 'repository_package.dart';
 
 /// An entry in package enumeration for APIs that need to include extra
 /// data about the entry.
 class PackageEnumerationEntry {
   /// Creates a new entry for the given package directory.
-  PackageEnumerationEntry(this.directory, {required this.excluded});
+  PackageEnumerationEntry(this.package, {required this.excluded});
 
-  /// The package's location.
-  final Directory directory;
+  /// The package.
```

Comment:
nit: not really a helpful comment.

Reply:
I added a warning about `excluded` to this, as making sure that's not missed seemed like the only useful thing to say here. (I have to say something, because the repo has the lint requiring comments on public fields.